website/source/docs/builders/triton.html.md

```diff
@@ -149,7 +152,7 @@ builder.
 
 ## Basic Example
 
-Below is a minimal example to create an joyent-brand image on the Joyent public
+Below is a minimal example to create a joyent-brand image on the Joyent public
```

Comment:
I'm not sure what a "joyent-brand" is.  This is an Illumos Zone-based image, or an "infrastructure VM."  This nit predates this PR but now seems like a good time to fix this.